sktime/transformations/series/adi_cv.py

```diff
@@ -59,6 +63,14 @@ class ADICVTransformer(BaseTransformer):
     features : list[str] | None (default = ['adi', 'cv2', 'class'])
         Specifies all of the feature values to be calculated
 
+    adi_trim_handling: string (default = pool)
+        Specifies the method for reconciling leading/trailing zeros in the series.
+        Allowable values are ('pool', 'trim', and 'ignore'), corresponding to the
+        following treatment of leading/trailing zeros:
+        pool => N Observations / N Non-Zero Observations
```

Comment:
use of the letter "N" is confusing, since a reader might think it corresponds to the same integer. How about you define the L and N as in the issue to avoid ambiguity?

Reply:
I saw the definitions for L and N. It definitely makes more sense now!